phoenix-pherf/src/main/java/org/apache/phoenix/pherf/util/PhoenixUtil.java

```diff
@@ -51,6 +51,7 @@
 import java.text.SimpleDateFormat;
 import java.util.ArrayList;
 import java.util.Collections;
+import java.util.HashMap;
```

Comment:
nit: redundant import

Reply:
Where's the redundancy? This was fixing a build failure from a missing import from a previous JIRA someone else pushed. (I've since fixed it as an addendum so this will go away when I rebase, I think.)